Content.Server/CharacterInfo/CharacterInfoSystem.cs

```diff
@@ -72,8 +80,19 @@ private void OnRequestCharacterInfoEvent(RequestCharacterInfoEvent msg, EntitySe
 
             // Get briefing
             briefing = _roles.MindGetBriefing(mindId);
+
+
+            // Gabystation change - bank
+            var station = _station.GetOwningStation(entity);
+            if (mind.NanoBankAccount is not null && TryComp<EconomyManagerComponent>(station, out var economy))
+            {
+                _bank.GetAccountPassword(economy, mind.NanoBankAccount ?? 0, true, out var password);
+                nanoBankBriefing = Loc.GetString("economy-character-info-briefing", ("number", mind.NanoBankAccount ?? 0), ("password", password));
```

Comment:
```suggestion
                _bank.GetAccountPassword(economy, mind.NanoBankAccount.Value, true, out var password);
                nanoBankBriefing = Loc.GetString("economy-character-info-briefing", ("number", mind.NanoBankAccount.Value), ("password", password));
```

Sabemos que esses valores são não-nulos. Menos confuso assim.